packages/safe-ds-eda/src/apis/historyApi.ts

```diff
@@ -220,3 +227,30 @@ const evaluateMessagesWaitingForTurn = function () {
     messagesWaitingForTurn = newMessagesWaitingForTurn;
     if (firstItemQueueChanged) evaluateMessagesWaitingForTurn(); // Only if first element was deployed we have to scan again, as this is only deployment condition
 };
+
+const updateTabOutdated = function (entry: ExternalHistoryEntry | InternalHistoryEntry): void {
+    if (entry.action === 'hideColumn' || entry.action === 'showColumn') {
+        tabs.update((state) => {
+            const newTabs = state.map((t) => {
+                if (t.type !== 'empty') {
+                    if (
+                        t.columnNumber === 'none' &&
+                        get(table)?.columns.find((c) => c.name === entry.columnName)?.type === 'numerical'
+                    ) {
+                        // UPDATE the if in case there are none column tabs that do not depend on numerical columns
+                        return {
+                            ...t,
+                            outdated: true,
+                        };
+                    } else {
+                        return t;
+                    }
+                } else {
+                    return t;
+                }
```

Comment:
```suggestion
                if (t.type !== 'empty' &&
                    t.columnNumber === 'none' &&
                    get(table)?.columns.find((c) => c.name === entry.columnName)?.type === 'numerical') {
                        // UPDATE the if in case there are none column tabs that do not depend on numerical columns
                    return {
                        ...t,
                        outdated: true,
                    };
                } else {
                    return t;
                }
```